website/docs/seo.mdx

```diff
@@ -124,15 +125,15 @@ export default function page() {
         <meta property="og:image" content="image.png" />
         <meta name="twitter:card" content="summary_large_image" />
         <link rel="preconnect" href="https://example.com" />
-        <script type="application/ld+json">
-          {JSON.stringify({
+        <StructuredData
+          structuredData={{
             '@context': 'https://schema.org/',
             '@type': 'Organization',
             name: 'Meta Open Source',
             url: 'https://opensource.fb.com/',
             logo: 'https://opensource.fb.com/img/logos/Meta-Open-Source.svg',
-          })}
-        </script>
+          }}
+        />
```

Comment:
Looks like Helmet doesn't support well custom components 🤪 

This doesn't work:

```tsx
      <Head>
        <CustomComp />
      </Head>
```

![CleanShot 2024-02-14 at 15 00 47](https://github.com/facebook/docusaurus/assets/749374/16227ea8-42b2-45af-9834-04e9dcac68be)

This also doesn't work 🤪 

```js
      <Head>
        <script
          data-seb-id="seb-id-2"
          type="application/ld+json"
          dangerouslySetInnerHTML={{
            __html: JSON.stringify({
              '@context': 'https://schema.org/',
              '@type': 'Organization',
              name: 'Meta Open Source',
              url: 'https://opensource.fb.com/',
              logo: 'https://opensource.fb.com/img/logos/Meta-Open-Source.svg',
            }),
          }}
        />
      </Head>
```